flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -1155,7 +1155,14 @@ def create_chargepoint_session_chart(
             },
         ],
     }
-
+    for idx, entry in enumerate(sensors_to_show):
+        title = entry.get("title")
+        if title == "Power flow by type":
+            sensors_to_show[idx]["sensors"] = [
+                sensor
+                for sensor in entry["sensors"]
+                if sensor.name == "charge points power"
```

Comment:
It looks to me like you are replacing an existing entry in sensors_to_show, keeping only the one(s) with the name "charge points power"?

That does not make sense to me or corresponds to the commit message for this. Nor does it have to do with prices, which is this PR's title.